openslides/agenda/models.py

```diff
@@ -269,7 +271,7 @@ def get_list_of_speakers(self, old_speakers_count=None, coming_speakers_count=No
                 speaker_dict['last_in_group'] = True
             list_of_speakers.append(speaker_dict)
 
-        # Parse actual speaker
+        # Parse current speaker
```

Comment:
I deleted this function, because it was not called anywhere. If it is still needed, then it has to be implemented in javascript.

But actual_speaker  occurred on another place, I fixed it there.